### DISCUSSION TITLE
Table Inline Columns Enhancement

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#5020] [Ideas]
I think following options and methods can be useful in Inline Columns

- `beforeAction()`
- `afterAction()`
- `successNotification()`
- `failedNotification()`
- Confirmation Action like Delete Button

**Comment by ManojKiranA** [comment]
on [discord](https://discord.com/channels/883083792112300104/883083792653381695/1057267456684593152) chat

ping @pxlrbt

**Comment by ManojKiranA** [comment]
Hey @pxlrbt is it possible after #4990

**Reply by pxlrbt** [reply]
You could override it yourself, but we could also add these as new methods.

**Reply by ManojKiranA** [reply]
May be as as temporary fix I will override this method.
